packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts

```diff
@@ -116,14 +117,26 @@ function resolveLastRecognizedIdentityMobileInner(
     window.addEventListener('locationchange', onLocationChange, { signal: cancel })
 }
 
+const getFirstSlug = () => {
```

Comment:
Why not compare the full URL?

Reply:
For example, https://twitter.com/VitalikButerin and https://twitter.com/VitalikButerin/with_replies both contain information of current visiting user